lms/static/js/certificates/factories/certificate_whitelist_factory.js

```diff
@@ -16,7 +16,7 @@
             return function(certificate_white_list_json, generate_certificate_exceptions_url,
                             certificate_exception_view_url, generate_bulk_certificate_exceptions_url){
 
-                var certificateWhiteList = new CertificateWhiteListCollection(JSON.parse(certificate_white_list_json), {
+                var certificateWhiteList = new CertificateWhiteListCollection(certificate_white_list_json, {
```

Comment:
Good catch. 

We only need to escape data once. This will also help in making templates consistent for Safe Template Marathon.